packages/nextjs/src/utils/mergeNextClerkPropsWithEnv.ts

```diff
@@ -8,9 +8,9 @@ export const mergeNextClerkPropsWithEnv = (props: Omit<NextClerkProviderProps, '
     publishableKey: props.publishableKey || process.env.NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY || '',
     clerkJSUrl: props.clerkJSUrl || process.env.NEXT_PUBLIC_CLERK_JS,
     clerkJSVersion: props.clerkJSVersion || process.env.NEXT_PUBLIC_CLERK_JS_VERSION,
-    proxyUrl: props.proxyUrl || process.env.NEXT_PUBLIC_CLERK_PROXY_URL || '',
-    domain: props.domain || process.env.NEXT_PUBLIC_CLERK_DOMAIN || '',
-    isSatellite: props.isSatellite || isTruthy(process.env.NEXT_PUBLIC_CLERK_IS_SATELLITE),
+    proxyUrl: props.proxyUrl || (process.env.NEXT_PUBLIC_CLERK_PROXY_URL as any), // added as any to cater issue with type resolution
+    domain: props.domain || (process.env.NEXT_PUBLIC_CLERK_DOMAIN as any), // added as any to cater issue with type resolution
+    isSatellite: props.isSatellite || (isTruthy(process.env.NEXT_PUBLIC_CLERK_IS_SATELLITE) as any), // added as any to cater issue with type resolution
```

Comment:
@panteliselef i had to do this in order for the `MultidomainAndOrProxy` types to pass. let's discuss it tomorrow and see if we can simplify the type or fix the current issues.